tests/_helpers.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+import mock
+
+try:
+    from opentelemetry import trace as trace_api
+    from opentelemetry.trace.status import StatusCanonicalCode
+
+    from opentelemetry.sdk.trace import TracerProvider, export
+    from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+        InMemorySpanExporter,
+    )
+
+    HAS_OPENTELEMETRY_INSTALLED = True
+except ImportError:
+    HAS_OPENTELEMETRY_INSTALLED = False
```

Comment:
Why cannot OpenTelemetry be installed? This adds a lot of checks `if HAS_OPENTELEMETRY_INSTALLED` in the following code. Is it possible we can get rid of this? 

If `OpenTelemetry` is not installed, the test will fail and I think this should be fine.

Reply:
OpenTelemetry can't be installed for python2.7, so as long as we're running those tests this needs to be here